data/transactions/logic/TEAL_opcodes.md

```diff
@@ -425,6 +425,10 @@ The notation J,K indicates that two uint64 values J and K are interpreted as a u
 | 61 | CreatedApplicationID | uint64 | v5  | ApplicationID allocated by the creation of an application (only with `itxn` in v5). Application mode only |
 | 62 | LastLog | []byte | v6  | The last message emitted. Empty bytes if none were emitted. Application mode only |
 | 63 | StateProofPK | []byte | v6  | 64 byte state proof public key commitment |
+| 64 | ApprovalProgramPages | []byte | v7  | Approval Program as an array of pages |
+| 65 | NumApprovalProgramPages | uint64 | v7  | Number of Approval Program pages |
+| 66 | ClearStateProgramPages | []byte | v7  | ClearState Program as an array of pages |
+| 67 | NumClearStateProgramPages | uint64 | v7  | Number of ClearState Program pages |
```

Comment:
I don't know why this didn't make it to the live page? https://developer.algorand.org/docs/get-details/dapps/avm/teal/opcodes/#txn-f

Is it just that it waits until the actual release on the 31st?

Reply:
Yes, I think that's only refreshed by hand. `vrf_verify` and `block` are also not there.

Things like `replace` which are v7 are only there because we must have refreshed it a month or so back when they were in master by these were not.

Reply:
Ok! Thanks :) I just integrated all of these features into Reach, so we'll be live the same time as you :)